src/assets/scripts/client/trafficGenerator/SpawnPatternCollection.js

```diff
@@ -111,6 +127,38 @@ export default class SpawnPatternCollection extends BaseCollection {
     }
 
     /**
+     * Gather a randomized list of departure `SpawnPatternModel` objects
+     * that will be created during the prSpawn.
+     *
+     * Clamps max at 5 patterns.
+     * Randomly selects departure patterns from a filtered list containing only departure patterns.
+     *
+     * This method should be called by the `SpawnScheduler` when preSpawning new
+     * aircraft on session start.
+     *
+     * @for SpawnPatternCollection
+     * @method getDepartureModelsForPreSpawn
+     * @return departureModelsForPreSpawn {array<SpawnPatternModel>}
+     */
+    getDepartureModelsForPreSpawn() {
+        const departureModelsForPreSpawn = [];
+        const minimumDeparturesToPreSpawn = 1;
+        const maximumDeparturesToPreSpawn = 5;
```

Comment:
as *the* number of aircraft to spawn. If it's really that many, maybe we only need ten minutes' worth.